### PR TITLE
made intakeCone also work for cubes

### PR DESCRIPTION
**Pull request by AlenaNguyen**
included an if statement to make intake speed change depending on the game piece

### CODE REVIEW

File: src/main/java/frc/robot/RobotPreferences.java [hunk 1]
```diff
@@ -137,7 +138,8 @@ public static final class prefElevator {
     public static final SN_DoublePreference elevatorMaxPos = new SN_DoublePreference("elevatorMaxPos", 10);
     public static final SN_DoublePreference elevatorMinPos = new SN_DoublePreference("elevatorMinPos", 0);
 
-    public static final SN_DoublePreference elevatorIntakingPos = new SN_DoublePreference("elevatorIntakingPos", 2);
+    public static final SN_DoublePreference elevatorIntakingPos = new SN_DoublePreference("elevatorConeIntakingPos",
```
Comment (TaylerUva) [inline src/main/java/frc/robot/RobotPreferences.java]: The variable name and key string do not match

**Review by TaylerUva — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -109,8 +110,11 @@ private void configureBindings() {
 
     // Operator
 
-    // Intake Cube (RB)
-    conOperator.btn_RightBumper.onTrue(new IntakeCone(subWrist, subIntake, subElevator));
+    // Intake Cone (RB)
+    conOperator.btn_RightBumper.onTrue(new IntakeGamePiece(subWrist, subIntake, subElevator, GamePiece.CONE));
+
+    // Intake Cube (LB)
+    conOperator.btn_RightBumper.onTrue(new IntakeGamePiece(subWrist, subIntake, subElevator, GamePiece.CUBE));
```
Comment (alicekuznetsov) [inline src/main/java/frc/robot/RobotContainer.java]: // Intake Cube (LB)
you typed btn_RightBumper

File: src/main/java/frc/robot/RobotPreferences.java [hunk 3]
```diff
@@ -137,7 +138,8 @@ public static final class prefElevator {
     public static final SN_DoublePreference elevatorMaxPos = new SN_DoublePreference("elevatorMaxPos", 10);
     public static final SN_DoublePreference elevatorMinPos = new SN_DoublePreference("elevatorMinPos", 0);
 
-    public static final SN_DoublePreference elevatorIntakingPos = new SN_DoublePreference("elevatorIntakingPos", 2);
+    public static final SN_DoublePreference elevatorIntakingPos = new SN_DoublePreference("elevatorIntakingPos",
```
Comment (alicekuznetsov) [inline src/main/java/frc/robot/RobotPreferences.java:135]: Could you add a unit to this? If there is no unit you don't need to

File: src/main/java/frc/robot/commands/IntakeGamePiece.java [hunk 4]
```diff
@@ -6,30 +6,41 @@
...
     this.subWrist = subWrist;
     this.subIntake = subIntake;
+    this.subElevator = subElevator;
+    this.gamepiece = gamepiece;
+
+    if (gamepiece == GamePiece.CONE) {
+      intakeSpeed = prefIntake.intakeConeSpeed.getValue();
+    } else if (gamepiece == GamePiece.CUBE) {
+      intakeSpeed = prefIntake.intakeCubeSpeed.getValue();
+    }
```
Comment (alicekuznetsov) [inline src/main/java/frc/robot/commands/IntakeGamePiece.java:40]: Could you add a fallback condition? I know it should never be anything else but redundancy doesn't hurt

**Review by alicekuznetsov — DISMISSED** [review]

**Review by TaylerUva — DISMISSED** [review]

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**